2019Labs/RHELSecurityLab/documentation/lab6_Audit.adoc

```diff
@@ -268,7 +268,9 @@ user and run the following commands:
 	[auditlab@audit ~]$ ping -c 1 127.0.0.1
 
 	[auditlab@audit ~]$ vi ~/project_tps_report.txt
-	(Type :q! to exit vi)
```

Comment:
For people that are not familiar with `vi`, this will cause distractions, imho. Is it possible to replace this part with simple echo `This is my TPS report`?

Reply:
One more nitpick - if we want to stay with `vi`, please explicitly mention *esc* needs to be pressed before `:wq!` can be used.